src/test/java/io/openliberty/tools/common/plugins/util/InstallFeatureUtilGetServerFeaturesTest.java

```diff
@@ -598,7 +598,8 @@ private void replaceIncludeLocation(String includeLocation) throws Exception {
      */
     @Test
     public void testIncludeDir() throws Exception {
-        replaceIncludeDir("includeDir");
+        // Note: Both the product code and test code end up converting Windows \ into /
+        replaceIncludeLocation("includeDir/"); 
```

Comment:
Should you test without the `/` and prove it doesn't get processed (and the error is logged)?

Reply:
Will add a test for this failing case